python/lsst/jointcal/jointcal.py

```diff
@@ -887,7 +887,7 @@ def loadData(self, dataRefs, associations, jointcalControl):
         """Read the data that jointcal needs to run. (Gen2 version)"""
         visit_ccd_to_dataRef = {}
         oldWcsList = []
-        bands = []
+        filters = []
```

Comment:
Same here for the filterLabel (I think) and this method doesn't have a proper docstring.

Reply:
If it's ok, I'm not going to bother adding a docstring since this is purely gen2.